libs/chiliui/chili/headers/util.lua

```diff
@@ -64,26 +64,10 @@ end
 
 function IsRelativeCoord(code)
 	local num = tonumber(code)
-
-	if (type(code) == "string") then
-		return true
-	elseif (num) and ((1/num) < 0) then
-		return true
-	else
+	if num and ((1/num) < 0) then
```

Comment:
I reasoned we're more likely to give an pixel constraint than a string constraint? In the map list, it helps.